config-server/src/main/resources/bootstrap-local.yml

```diff
@@ -0,0 +1,7 @@
+spring:
+  application:
+    name: config-server
+  profiles: local
```

Comment:
why do you need the local here?
its already the local file

Reply:
For docker-compose, for some reason doesn't recognize the profile by the name in bootstrap